src/pcapi/admin/custom_views/suspend_fraudulent_users_by_ids.py

```diff
@@ -0,0 +1,65 @@
+import csv
+from io import TextIOWrapper
+
+from flask import flash
+from flask import request
+from flask_admin import expose
+from flask_admin.form import SecureForm
+from flask_login import current_user
+from werkzeug import Response
+from wtforms import FileField
+
+from pcapi.admin.base_configuration import BaseCustomAdminView
+from pcapi.scripts.suspend_fraudulent_beneficiary_users import suspend_fraudulent_beneficiary_users_by_ids
+
+
+ALLOWED_EXTENSIONS = {"csv"}
+
+
+def allowed_file(filename):
+    return "." in filename and filename.rsplit(".", 1)[1].lower() in ALLOWED_EXTENSIONS
+
+
+class SuspendFraudulentUsersByIdsForm(SecureForm):
+    user_ids_csv = FileField("Importer un fichier CSV contenant une seule colonne des utilisateurs à suspendre")
+
+
+class SuspendFraudulentUsersByUserIdsView(BaseCustomAdminView):
+    @expose("/", methods=["GET", "POST"])
+    def search(self) -> Response:
+        form = SuspendFraudulentUsersByIdsForm()
+        fraudulent_users = []
+        nb_cancelled_bookings = 0
+        if request.method == "POST":
+            if not self.check_super_admins():
+                flash("Vous n'avez pas les droits pour effectuer cette opération", "error")
+            else:
+                form = SuspendFraudulentUsersByIdsForm(request.form)
+                user_ids_csv_file = request.files["user_ids_csv"]
+                if user_ids_csv_file and allowed_file(user_ids_csv_file.filename):
+                    user_ids_csv_file = TextIOWrapper(user_ids_csv_file, encoding="ascii")
+                    csv_rows = csv.reader(user_ids_csv_file)
+                    user_ids = []
+                    for row in csv_rows:
+                        if row[0].isdigit():
+                            user_ids.append(int(row[0]))
```

Comment:
(style / question de goût) Variante avec une _list comprehension_, qui me semble plus compacte tout en restant relativement lisible :

```suggestion
                    user_ids = [int(row[0]) for row in csv_rows if row[0].is_digit()]
```